### PR TITLE
Added IQuestionReportRepository file and QuestionReportRepository file

### PR DESCRIPTION
**Pull request by faranasj**

### CODE REVIEW

**Review by absyinka — CHANGES_REQUESTED** [review]
Both interface and implementing class does not implement /inherit respective interface/class.

Do make changes as pointed above

**Review by NurudeenAdeyemi — COMMENTED** [review]
The interface class should inherit the IRepository with the type parameter replaced by the actual class. The implementation name is not correct and should also inherit the baseRepository and the Interface of its own as well as constructor injecting the context class

File: Repository/Implementations/QuestionReportRepository.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace IdealDiscuss.Repository.Implementations
+{
+    public class QuestionReport
```
Comment (NurudeenAdeyemi) [inline Repository/Implementations/QuestionReportRepository.cs]: This should be QuestionReportRepository

Comment (NurudeenAdeyemi) [inline Repository/Implementations/QuestionReportRepository.cs]: and Inherit BaseRepository<QuestionReport>, IQuestionReportRepository

File: Repository/Interfaces/IQuestionReportRepository.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿namespace IdealDiscuss.Repository.Interfaces
+{
+    public interface IQuestionReportRepository
```
Comment (NurudeenAdeyemi) [inline Repository/Interfaces/IQuestionReportRepository.cs]: where is the class its inheriting

**Review by faranasj — COMMENTED** [review]
Please ignore this commit, I made a mistake

**Review by NurudeenAdeyemi — APPROVED** [review]

**Review by absyinka — APPROVED** [review]

**Pull request merged**